src/content/en/updates/2018/03/webauthn-credential-management.md

```diff
@@ -0,0 +1,176 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API 
```

Comment:
Upon discussing with Mike, let's make the title less depressing:

"Credential Management API Feature Detection Check-up"